extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -161,10 +161,10 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
-    /** @private {?../../amp-story/0.1/amp-story.AmpStory} */
+    /** @private {?../../amp-story/1.0/amp-story.AmpStory} */
```

Comment:
Remnant of moving from `0.1` to `1.0`? Is this change required as part of moving to a new Closure Compiler or just convenient to make in this PR?

Reply:
Yes, looks like a remnant that wasn't taken care of earlier. Without this change:
```js
extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js:263: ERROR - assignment to property navigationState_ of AmpStoryAutoAds
found   : NavigationState$$module$extensions$amp_story$0_1$navigation_state
required: (NavigationState$$module$extensions$amp_story$1_0$navigation_state|null)
    this.navigationState_ = this.ampStory_.getNavigationState();
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js:732: ERROR - actual parameter 1 of AmpStoryAutoAds$$module$extensions$amp_story_auto_ads$0_1$amp_story_auto_ads.prototype.nextPageNoAd_ does not match formal parameter
found   : AmpStoryPage$$module$extensions$amp_story$0_1$amp_story_page
required: (AmpStoryPage$$module$extensions$amp_story$1_0$amp_story_page|null)
        || this.nextPageNoAd_(pageBeforeAd)
                              ^^^^^^^^^^^^

extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js:750: ERROR - Property getPageIndexById never defined on AmpStory
    const pageNumber = this.ampStory_.getPageIndexById(pageBeforeAdId);
                                      ^^^^^^^^^^^^^^^^
```